docs/templating/twig_function_reference/other_twig_filters.md

```diff
@@ -1,5 +1,20 @@
 # Other Twig filters
 
+### `ibexa_user_get_current()`
+
+`ibexa_user_get_current()` returns the User object of the current user.
```

Comment:
Specifically, what type of `User` object is returned? What is it's fully qualified class name?